src/commonTest/kotlin/fr/acinq/lightning/io/peer/PeerTest.kt

```diff
@@ -62,9 +62,9 @@ class PeerTest : LightningTestSuite() {
         val bob = buildPeer(this, TestConstants.Bob.nodeParams, TestConstants.Bob.walletParams)
 
         // start Init for Alice
-        alice.send(MessageReceived(Init(features = TestConstants.Bob.nodeParams.features)))
+        alice.send(MessageReceived(Init(features = TestConstants.Bob.nodeParams.features), connectionId = 0))
```

Comment:
I intended to remove the default too, but also switch the order of args to put `connectionId` first.